app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
+  self.per_page = 10
```

Comment:
Second that. Maybe put in a file that include public variables that you can take later. Maybe this could help: http://guides.rubyonrails.org/configuring.html#using-initializer-files